### PR TITLE
Add option for known variance case

### PR DESCRIPTION
**Pull request by JeffFessler**
For some comparisons it is helpful to examine the case where the variances are known, and this tiny change provides an option to easily support that.
I also changed the tests to test Julia "1" (which is whatever the current version is).

### CODE REVIEW

**Comment by JeffFessler** [comment]
@dahong67 do you really want to keep testing Julia 1.2 and 1.4?  Julia 1.6 is now the long-term support release, so personally I would suggest we just test 1.6, "1" and nightly.  But it's OK to keep testing 1.2 and 1.4 if you want.

**Review by dahong67 — APPROVED** [review]
Looks good to me! Agreed about supporting Julia v1.6.

I've made a few minor tweaks, added tests, and will merge.

File: test/runtests.jl [hunk 1]
```diff
@@ -65,6 +65,13 @@ n, v = (40, 10), (4, 1)
         end
         @test Mr !== heppcat(Yb,k,T)
         @test Mr == heppcat(Yb,k,T)
+        
+        Mr = homppca(Yb,k)
+        for _ in 1:T
+            updateF!(Mr,Yb,ExpectationMaximization())
+        end
+        @test Mr !== heppcat(Yb,k,T;vknown=true)
+        @test Mr == heppcat(Yb,k,T;vknown=true)
```
Comment (JeffFessler) [inline test/runtests.jl:74]: should one of these be `false` ??

Comment (dahong67) [reply test/runtests.jl:74]: No, the first is just checking that the objects are not identical (perhaps not necessary). The second checks that their values match.

The preceding two checks (for the default `vknown=false`) do the same.

Comment (JeffFessler) [reply test/runtests.jl:74]: oh, doh!

**Comment by dahong67** [comment]
Merged!

**Comment by dahong67** [comment]
Merged! 🎊

**Comment by dahong67** [comment]
Merged - thanks @JeffFessler! 🎊

**Pull request merged**